features/docker.feature

```diff
@@ -19,7 +19,9 @@ Feature: Build docker images with ua services
             apt-get update \
             && apt-get install --no-install-recommends -y ubuntu-advantage-tools ca-certificates \
 
-            && dpkg -i /ua.deb \
+            && dpkg -i /ua.deb | true \
```

Comment:
```suggestion
            && dpkg -i /ua.deb || true \
```

Reply:
actually, will using `||` mess with the overall command flow? since the whole thing is `&&`'d together?

Maybe keeping `|` is best - up to you

Reply:
Fair enough, I will change it and merge the PR